### PR TITLE
Adding Mentor

### PR DESCRIPTION
**Pull request by kartikeya23**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by kartikeya23** [comment]
#36 
Added a mentor...

File: _data/mentors.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@
   github: jigyasa-grover
   image: jigyasa-grover.jpg
   facebook: grover.jigyasa
-  twitter: jigyasa_grover
+-   twitter: jigyasa_grover
```
Comment (abishekvashok) [inline _data/mentors.yml:55]: please don't include -, jekyll will parse it as a new mentor

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Also make the image to be in 240x240px

File: _data/mentors.yml (hunk 1 above)
Comment (kartikeya23) [reply _data/mentors.yml:55]: How do I change this? Do I need to make a new PR?

Comment (akshatnitd) [reply _data/mentors.yml:55]: Make the required changes.
Then, amend the commit and force push the branch.

```
git add _data/mentors.yml
git commit --amend
git push origin <branch-name> -f
```

**Pull request closed without merging**